wg-naming/governance.md

```diff
@@ -0,0 +1,85 @@
+# WG Naming governance
+
+## Uphold project values
+
+WG Naming upholds the Kubernetes project's [core values](./values.md).
+
+### Inclusive is better than exclusive
+
+> Broadly successful and useful technology requires different perspectives and skill sets which can only be heard in a welcoming and respectful environment.
+
+This document covers some specific ways that respect and welcome look like in WG Naming.
```

Comment:
This sentence doesn't appear to be grammatically clear/correct

Reply:
It is correct, but I can make it clearer.

Reply:
Also—I am on vacation for the next two weeks, starting yesterday. I will address this feedback, and ask that you consider making any further feedback iterable by another WG Naming lead in a subsequent PR.